src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/avoiddoublebraceinitialization/InputAvoidDoubleBraceInitialization.java

```diff
@@ -11,12 +11,12 @@
 
 public class InputAvoidDoubleBraceInitialization {
     void m() {
-        ArrayList<?> list = new ArrayList<Object>() { // violation
+        ArrayList<?> list = new ArrayList<Object>() { // violation 'Avoid double brace initialization.'
```

Comment:
Put it on line above and make it like 
`// violation below 'Avoid double brace initialization.'`